R/datasets.R

```diff
@@ -39,3 +39,10 @@
 #' @description The region codes for JHU
 #' @return A tibble of region codes and related information.
 "JHU_codes"
+
+#' Table of available datasets along with level and other information.
+#' Rendered from the individual R6 class objects included in this package.
+#'
+#' @description Available datasets
+#' @return A tibble of available datasets and related information.
+"all_country_data"
```

Comment:
Does this show up in the `pkgdown` reference? I think not as I think that is look for `_codes` in a name to declare it as a data set. Could add another tag (like utilities etc) or add it manually?